metadata-editor/app/lib/EditsStore.scala

```diff
@@ -1,6 +1,62 @@
 package lib
 
-import com.gu.mediaservice.lib.aws.DynamoDB
-import com.gu.mediaservice.model.Edits
+import com.gu.mediaservice.model.{Edits, ImageMetadata, UsageRights}
+import com.gu.mediaservice.lib.aws.{DynamoDB, DynamoHelpers}
+import org.scanamo.{DynamoFormat, DynamoReadError, ScanamoAsync, Table}
+import org.scanamo.generic.semiauto._
+import org.scanamo.generic.auto.genericDerivedFormat
+import org.scanamo.generic.semiauto.deriveDynamoFormat
+import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient
+import cats.implicits._
+import org.joda.time.DateTime
+import org.scanamo.syntax._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import play.api.libs.json.JsValue
+import ImageMetadata._
+import UsageRights._
+
+class EditsStore(config: EditsConfig) extends DynamoDB[Edits](config, config.editsTable, Some(Edits.LastModified)) with DynamoHelpers {
+  val tableName = config.editsTable
+  lazy val dynamoClient: DynamoDbAsyncClient = config.withAWSCredentialsV2(DynamoDbAsyncClient.builder()).build()
+  implicit val dateTimeFormat: Typeclass[DateTime] =
+    DynamoFormat.coercedXmap[DateTime, String, IllegalArgumentException](DateTime.parse, _.toString)
+  implicit val editsFormat: DynamoFormat[Edits] = deriveDynamoFormat[Edits]
+
+  private lazy val editsTable = Table[Edits](tableName)
+
+  def getV2(id: String): Future[Option[Edits]] =
```

Comment:
Calling these v2 for now, as there are still some parts of the code base that use the method in the dynamo trait. We can rename when we are able to deprecate the whole trait.